dockstore-webservice/src/main/java/io/dockstore/webservice/core/ExtendedUserData.java

```diff
@@ -31,7 +31,7 @@
 
     public ExtendedUserData(User user, PermissionsInterface authorizer) {
         Hibernate.initialize(user.getEntries());
-        this.canChangeUsername = user.getEntries().stream().noneMatch(Entry::getIsPublished) && !authorizer.isSharing(user);
+        this.canChangeUsername = user.getEntries().stream().noneMatch(Entry::getIsPublished) && !authorizer.isSharing(user) && user.getOrganizations().size() == 0;
```

Comment:
As food for thought, this method is pretty slow, relatively speaking; it takes .8 seconds to execute on staging for me, and 3 seconds on my admittedly much slower local box, and it shows up in the UI as a noticeable delay after logging in.

We should consider making this faster perhaps with some new NamedQueries...

But I will log that separately.